certsubmitter/certsubmitter.go

```diff
@@ -44,9 +44,8 @@ const logStr = "Certificate Submitter"
 // the Log returns.
 func Run(ctx context.Context, lc *client.LogClient, ca *certgen.CA, sv *ct.SignatureVerifier, st storage.APICallWriter, l *ctlog.Log, period time.Duration) {
 	glog.Infof("%s: %s: started with period %v", l.URL, logStr, period)
-
 	schedule.Every(ctx, period, func(ctx context.Context) {
-		chain, sct, err := issueAndSubmit(ctx, lc, ca, st, l)
+		chain, sct, err := issueAndSubmit(ctx, lc, ca, st, l, false /* isPreSubmit */)
```

Comment:
You could go for it and use the random function you had before, that's simple enough to understand now.

Reply:
Let's not get ahead of ourselves.)

Reply:
In that case, you should change the PR description?

Reply:
Done